lib/Jsrt/ChakraCore.h

```diff
@@ -679,5 +679,42 @@ CHAKRA_API
         _In_ JsPropertyIdRef propertyId,
         _Out_ bool *hasOwnProperty);
 
+/// <summary>
+///     Write JS string value into char string buffer without a null terminator
```

Comment:
Ah yes, that's great. Also clarify that UTF16 encoded codepoints are being truncated in this case. Actually, what happens in the case of a surrogate pair?